requirements-dev.txt

```diff
@@ -26,73 +26,67 @@ pytest-codspeed==2.2.1
 
 # Testing dependencies
 lxml==5.3.0
-dask==2023.5.0; python_version == '3.8'
-dask[dataframe]==2024.4.1; python_version >= '3.9'
-numpy; python_version < '3.9'
-numpy==1.26.2; python_version >= '3.9'
-pandas==2.0.3; python_version == '3.8'
-pandas==2.1.3; python_version >= '3.9'
+dask[dataframe]==2024.4.1
+numpy==1.26.2
+pandas==2.1.3
 xxhash>=3.0.0
 Pillow==10.4.0
 opencv-python==4.10.0.84
 tiktoken==0.7.0
 duckdb==1.1.2
 
 # Pyarrow
-pyarrow==16.0.0; python_version >= '3.9'
-pyarrow==15.0.0; python_version < '3.9'
+pyarrow==16.0.0
 # Ray
-ray[data, client]==2.10.0; python_version == '3.8'
-ray[data, client]==2.34.0; python_version >= '3.9'
+ray[data, client]==2.34.0
 
 # Lance
-lancedb>=0.6.10; python_version >= '3.8'
+lancedb>=0.6.10
 
 #Iceberg
-pyiceberg==0.7.0; python_version >= '3.8'
-tenacity==8.2.3; python_version >= '3.8'
+pyiceberg==0.7.0
+tenacity==8.2.3
 
 # Delta Lake
 deltalake==0.5.8; platform_system == "Windows"
-deltalake==0.18.2; platform_system != "Windows" and python_version < '3.9'
-deltalake==0.19.2; platform_system != "Windows" and python_version >= '3.9'
+deltalake==0.18.2; platform_system != "Windows" and python_version < '3.10'
```

Comment:
We shouldn't need to update these right?

Reply:
I made these changes due to some dependency conflict before, maybe it works lemme check